common/types.go

```diff
@@ -245,9 +245,14 @@ type ObjectMetadata struct {
 	FieldsMap map[string]string
 }
 
+type PostAuthInfoParams struct {
```

Comment:
Ideally we can retrieve the access token from the authenticated HTTP client that's passed into the Connector. Adding an extra parameter here will make it hard to generalize this PostAuthInfo method for other providers. Can you see if that would be possible?

Reply:
Alternatively, is there another Hubspot API method that doesn't require a token to be passed in?